Compile error

```
git clone --recursive https://github.com/vgteam/vg.git
cd vg
make get-deps
. ./source_me.sh && make
```
```
In file included from /home/ppandey/sandbox/code/vg/include/gbwtgraph/gbwtgraph.h:9:0,
                 from gbwtgraph.cpp:1:
/home/ppandey/sandbox/code/vg/include/gbwtgraph/utils.h:4:10: fatal error: handlegraph/handle_graph.hpp: No such file or directory
 #include <handlegraph/handle_graph.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Makefile:45: recipe for target 'gbwtgraph.o' failed
make[1]: *** [gbwtgraph.o] Error 1
make[1]: Leaving directory '/home/ppandey/sandbox/code/vg/deps/gbwtgraph'
Makefile:374: recipe for target 'lib/libgbwtgraph.a' failed
make: *** [lib/libgbwtgraph.a] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -367,7 +367,7 @@ endif
 
 $(INC_DIR)/gbwtgraph/gbwtgraph.h: $(LIB_DIR)/libgbwtgraph.a
 
-$(LIB_DIR)/libgbwtgraph.a: $(LIB_DIR)/libgbwt.a $(LIB_DIR)/libsdsl.a $(wildcard $(GBWTGRAPH_DIR)/*.cpp) $(wildcard $(GBWTGRAPH_DIR)/include/gbwtgraph/*.h)
+$(LIB_DIR)/libgbwtgraph.a: $(LIB_DIR)/libgbwt.a $(LIB_DIR)/libsdsl.a $(LIB_DIR)/libhandlegraph.a $(wildcard $(GBWTGRAPH_DIR)/*.cpp) $(wildcard $(GBWTGRAPH_DIR)/include/gbwtgraph/*.h)
 ifeq ($(shell uname -s),Darwin)
 	+. ./source_me.sh && cp -r $(GBWTGRAPH_DIR)/include/gbwtgraph $(CWD)/$(INC_DIR)/ && cd $(GBWTGRAPH_DIR) && AS_INTEGRATED_ASSEMBLER=1 $(MAKE) $(FILTER) && mv libgbwtgraph.a $(CWD)/$(LIB_DIR)
 else

